go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/Dynatrace/dynatrace-operator
 go 1.20
 
 require (
+	github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137
```

Comment:
I'm a bit concerned about introducing new package for 1 single usage.
Also after checikng package, it doesn't have proper tests. 
Do it make sense to remove it and a little bit redo it on our own? 

Go proverb “a little copying is better than a little dependency” is to be careful when bringing new dependencies to our programs.

Reply:
cc @0sewa0

Reply:
true- I just used it as it was already there and I wanted to change as little as possible...

Reply:
read through the usage again and replaced it with 1024 * 1024- not that much of a change and as it is in test code I think worth it to remove a dependency

Reply:
@wepudt thanks!